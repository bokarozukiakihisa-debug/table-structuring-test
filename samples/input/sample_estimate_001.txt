（ここに OCRからのテキストや表抽出を仮置き）
材料費 10個 × 500 = 5,000
加工費 2式 × 3,000 = 6,000
小計 11,000
消費税 1,100
合計 12,100
