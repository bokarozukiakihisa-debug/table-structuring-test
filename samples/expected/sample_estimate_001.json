{
  "明細": [
    {"項目": "材料費", "数量": 10, "単価": 500, "金額": 5000},
    {"項目": "加工費", "数量": 2, "単価": 3000, "金額": 6000}
  ],
  "小計": 11000,
  "消費税": 1100,
  "合計": 12100
}
